app/identifiers/register/trustees/company/CompanyEmailId.scala

```diff
@@ -38,7 +38,7 @@ object CompanyEmailId {
     override def row(id: CompanyEmailId)(changeUrl: String, userAnswers: UserAnswers): Seq[AnswerRow] = {
       def trusteeName(index: Int) = userAnswers.get(CompanyDetailsId(index)).fold(messages("messages__theTrustee"))(_.companyName)
       def label(index:Int) = messages("messages__common_email__heading", trusteeName(index))
-      def hiddenLabel(index:Int) = Some(messages("messages__common_trustee_company_email__visually_hidden_change_label", trusteeName(index)))
+      def hiddenLabel(index:Int) = Some(messages("messages__common_email__visually_hidden_change_label", trusteeName(index)))
```

Comment:
These don't seem to need to be functions (Int => String). We can just have on line 39:
```
val trusteeName = userAnswers.get(CompanyDetailsId(id.index)).fold(messages("messages__theTrustee"))(_.companyName)
```
and then here becomes
```
val hiddenLabel = Some(messages("messages__common_email__visually_hidden_change_label", trusteeName))```